libvmi/driver/xen/xen_events.c

```diff
@@ -668,6 +669,9 @@ void xen_events_destroy_new(vmi_instance_t vmi)
     rc = xen->libxcw.xc_monitor_write_ctrlreg(xch, dom, VM_EVENT_X86_CR4, false, false, false);
     rc = xen->libxcw.xc_monitor_write_ctrlreg(xch, dom, VM_EVENT_X86_XCR0, false, false, false);
     rc = xen->libxcw.xc_monitor_software_breakpoint(xch, dom, false);
+    rc2 = xen_set_guest_requested_event(vmi, 0);
```

Comment:
@tklengyel any negative implications that you can see here w/ ignoring the rc of any of these calls, like anything that could leave a guest in an odd state?  I understand why the rc variables are just overwritten most of the time, especially given that this function could be considered a best-effort cleanup/error path anyway.